src/contradictory_claims/cli.py

```diff
@@ -92,7 +94,48 @@ def main(train, report):
     covid19_filt_section_df = covid19_df.loc[covid19_df.section.str.lower().isin(section_list)]
 
     # Clean the text to keep only meaningful sentences
-    covid19_clean_df = clean_text(covid19_filt_section_df)  # noqa: F841
+    # and merge sentences belonging to each section of each paper into contiguous text passages
+    covid19_clean_df = clean_text(covid19_filt_section_df)
+
+    # Merge all sentences belonging to each section of each paper into contiguous text passages
+    covid19_merged_df = merge_section_text(covid19_clean_df)
+
+    # Construct regex match pattern for drug terms
+    # Flank drug terms with white space for accurate match
+    with open(drug_lex_path) as f:
+        drug_terms = f.read().splitlines()
+    drug_terms.append('acei/arb')  # TODO: Add this to the main lexicon file
+    drug_terms_pattern = construct_regex_match_pattern(drug_terms, 'flank_white_space')
+
+    # Filter to sections where section text contains drug terms
+    covid19_drugs_section_df = filter_section_with_drugs(covid19_merged_df,  # noqa: F841
+                                                         drug_terms, drug_terms_pattern)
+
+    # TODO: Replace with claim extraction code
+    claims_df = covid19_drugs_section_df
+
+    # Separate papers with at least 1 claim from those with no claims
+    no_claims_cord_uid = set(claims_df.loc[claims_df.claim_flag == 0, 'cord_uid'])\
+        - set(claims_df.loc[claims_df.claim_flag == 1, 'cord_uid'])
+    claims_data = claims_df.loc[claims_df.claim_flag == 1, :].copy().reset_index(drop=True)
+    no_claims_data = claims_df.loc[claims_df.cord_uid.isin(no_claims_cord_uid), :]\
+                              .copy().reset_index(drop=True)
+
+    # For papers with no claims, tokenize section text to sentences
+    # and append to claims
+    # This is because when no claims are identified, we want to consider all sentences
+    # rather than ignoring the paper altogether
+    no_claims_data = tokenize_section_text(no_claims_data)
+    claims_data = claims_data.append(no_claims_data).reset_index(drop=True)
```

Comment:
Yeah I know, been thinking about that myself. For v31, we had 30% of papers with no claims out of a total of 2.5k papers. This is what I'd asked Harsh to help investigate -  whether the papers classified as having no claims actually have no claims. I think that 30% is not too large a proportion to cause too much noise (since we're also filtering to sentences containing the drug term) but at the same time not too small for completely discarding the papers.